mne/gui/_coreg.py

```diff
@@ -746,6 +747,11 @@ def _configure_picking(self):
             self._on_pick
         )
 
+    def _configure_legend(self):
+        mri_fids_legend_actor = self._renderer.legend(
+            labels=[("LPA", "red"), ("Nasion", "green"), ("RPA", "blue")])
```

Comment:
can you avoid redefining here the colors of fiducials and define each color in only one place? besides can you use (0, 1, 0) for nasion in legend so it's more readable.